client/app/utils/i18n.js

```diff
@@ -13,35 +13,33 @@
 
 import { bind } from 'lodash';
 
-function isServer() {
-  return typeof window === 'undefined';
-}
-
-if (isServer()) {
-
-  // Initialize global.I18n
-  // In browser we initialize this in a script-tag manually
-  global.I18n = {}; // eslint-disable-line no-undef
-
-  // Load the translation bundle in the global.I18n variable.
-  // In browser the bundle is loaded in a separate script-tag.
-
-  try {
-    // The translation bundle will be loaded to the global I18n
-    // variable. Initialize the variable here.
-    require('../i18n/all.js'); // eslint-disable-line no-undef
-  } catch (e) {
-    console.warn("Can't load language bundle all.js"); // eslint-disable-line no-console
-  }
-}
+// Initialize global.I18n
+// This might be initialized already, because in browser
+// we do initialize this before the translations get loaded
+// in a separate script tag
+global.I18n = global.I18n || {}; // eslint-disable-line no-undef
```

Comment:
I get the same error when I try `global` in browser console.

But when I add this to the `i18n.js` file:

![screen shot 2016-05-16 at 11 47 41](https://cloud.githubusercontent.com/assets/429876/15284781/310ba6a4-1b5c-11e6-885f-430ed729d0b4.png)

I see this in the console:

![screen shot 2016-05-16 at 11 47 30](https://cloud.githubusercontent.com/assets/429876/15284792/37c98c90-1b5c-11e6-8bf8-497fcfb438d2.png)